monitoring/snippets/v3/cloud-client/snippets.py

```diff
@@ -18,10 +18,20 @@
 import time
 import uuid
 
+from google.api import metric_pb2  # type: ignore
+from google.api import monitored_resource_pb2  # type: ignore
+from google.cloud.monitoring_v3.services.metric_service import pagers
+
+
 PROJECT_ID = os.environ["GOOGLE_CLOUD_PROJECT"]
 
 
-def create_metric_descriptor(project_id):
+def create_metric_descriptor(project_id: str) -> None:
```

Comment:
Can you please redo the sample to return a value and use it in the test instead of output parsing?

Reply:
done